### PR TITLE
vioscreen comparison notebooks

### PR DESCRIPTION
**Pull request by serenejiang**

### CODE REVIEW

File: ipynb/vioscreen_comparisons.ipynb [hunk 1]
```diff
@@ -0,0 +1,1009 @@
+{
```
Comment (wasade) [inline ipynb/vioscreen_comparisons.ipynb:1]: Could you add some documentation cells to guide a reader as to what's going on?

File: ipynb/vioscreen_comparisons.ipynb [hunk 2]
```diff
@@ -0,0 +1,1009 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mapping = pd.read_csv('viscreen_subsample_mapping.txt', sep='\\t') "
```
Comment (wasade) [inline ipynb/vioscreen_comparisons.ipynb]: can you clarify where this file originated from? @jwdebelius do you know?

File: ipynb/vioscreen_comparisons.ipynb [hunk 3]
```diff
@@ -0,0 +1,1009 @@
...
+       "      <th>vioscreen_xylitol</th>\n",
+       "      <th>vioscreen_zinc</th>\n",
+       "      <th>faiths_pd_1250</th>\n",
+       "      <th>shannon_1250</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>10317.000040</td>\n",
```
Comment (wasade) [inline ipynb/vioscreen_comparisons.ipynb]: i'm concerned that these sample IDs are truncated. can this notebook be rerun using data that are not truncated to verify the results dont change?

**Review by wasade — CHANGES_REQUESTED** [review]

File: ipynb/vioscreen_comparisons.ipynb (hunk 2 above)
Comment (jwdebelius) [reply ipynb/vioscreen_comparisons.ipynb]: This was the map we filtered to use only the vioscreen samples, right? Can we do that filtering here, in the notebook?

File: ipynb/vioscreen_comparisons.ipynb (hunk 1 above)
Comment (serenejiang) [reply ipynb/vioscreen_comparisons.ipynb:1]: sure! Just Add in the new version.

File: ipynb/vioscreen_comparisons.ipynb (hunk 2 above)
Comment (serenejiang) [reply ipynb/vioscreen_comparisons.ipynb]: Yes, Justine. You are right. I also include the detailed code for filtering in the new notebook.

File: ipynb/vioscreen_comparisons.ipynb (hunk 3 above)
Comment (serenejiang) [reply ipynb/vioscreen_comparisons.ipynb]: In fact, I didn't use the Sample ID in the analysis. But I just re-run the result to make sure it is not affected.

**Review by wasade — APPROVED** [review]

**Pull request merged**